api/src/main/java/uk/gov/hmcts/payment/api/controllers/PaymentController.java

```diff
@@ -121,6 +136,35 @@ public PaymentsResponse retrievePayments(@RequestParam(name = "start_date", requ
         }
     }
 
+    @ApiOperation(value = "Update payment status by payment reference", notes = "Update payment status by payment reference")
+    @ApiResponses(value = {
+        @ApiResponse(code = 204, message = "No content"),
+        @ApiResponse(code = 404, message = "Payment not found")
+    })
+    @PaymentExternalAPI
+    @PatchMapping("/payments/{reference}/status/{status}")
+    @Transactional
```

Comment:
We need to agree on Transactional boundary in the whole project. I prefer rest layer should be independent of DB transactions.